ext/native/gfx_es2/gl3stub.h

```diff
@@ -504,8 +504,10 @@ extern GL_APICALL void           (* GL_APIENTRY glBindFragDataLocationEXT) (GLui
 extern GL_APICALL GLint          (* GL_APIENTRY glGetProgramResourceLocationIndexEXT) (GLuint program, GLenum programInterface, const GLchar *name);
 extern GL_APICALL GLint          (* GL_APIENTRY glGetFragDataIndexEXT) (GLuint program, const GLchar *name);
 
+#ifdef GL_EXT_buffer_storage
```

Comment:
This doesn't really make sense to me though - for other platforms, like probably Android, this will mean that we never use this extension.  Worse, we'll detect it, but then silently not call this function, so buffers will be used but broken.

-[Unknown]

Reply:
Ah, got it, as you pointed out Android and others do define this.  I guess we end up with the same symbol type.

-[Unknown]